ruddock/routes.py

```diff
@@ -17,6 +17,7 @@ def home():
 @login_required()
 def show_info():
   """Shows info page on door combos, printers, etc."""
+  # TODO - if desired, add excomm flag to show/hide ExComm Materials link
```

Comment:
This todo should be implemented before the excomm link is added to the website, so that the link is only accessible by excomm members. Could be accomplished as the comment suggests by showing/hiding the link based on whether the logged in user is part of excomm. Or, could create a separate url path to this page for excomm members that includes the link. Either way would be fine.